### PR TITLE
EXLM-1502 Adobe Account card

### PR DESCRIPTION
**Pull request by ruhisingh1**
Please always provide the Jira Issue your PR is for, as well as test URLs where your change can be observed (before and after):

Jira ID: EXLM-1502

Test URLs:

- Before: https://main--exlm--adobe-experience-league.hlx.live
- After: https://feature-exlm-1502--exlm--adobe-experience-league.hlx.live

### CODE REVIEW

File: blocks/adobe-account-card/adobe-account-card.css [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    align-items: center;
+    gap: 16px;
+}
+
+.adobe-account-card .avatar {
+    width: 64px;
+    height: 64px;
+    border-radius: 50%;
+    background-color: var(--non-spectrum-navy-blue);
+    color: var(--background-color);
```
Comment (noormd08) [inline blocks/adobe-account-card/adobe-account-card.css:57]: Pls check whether we have diff variable for this

File: blocks/adobe-account-card/adobe-account-card.css [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    letter-spacing: 0px;
+    color: var(--text-color);
+    margin-bottom: 4px;
+}
+
+.adobe-account-card .user-info .email,
+.adobe-account-card .user-info .company {
+    font-size: var(--spectrum-font-size-100);
+    font-weight: normal;
+    letter-spacing: 0px;
```
Comment (noormd08) [inline blocks/adobe-account-card/adobe-account-card.css:77]: Remove letter spacing if we are not overriding.

File: blocks/adobe-account-card/adobe-account-card.css (hunk 1 above)
Comment (ruhisingh1) [reply blocks/adobe-account-card/adobe-account-card.css]: Done

File: blocks/adobe-account-card/adobe-account-card.css (hunk 2 above)
Comment (ruhisingh1) [reply blocks/adobe-account-card/adobe-account-card.css]: Done

**Review by noormd08 — APPROVED** [review]

**Pull request merged**